src/lib/snack-bar/snack-bar.ts

```diff
@@ -64,6 +62,10 @@ export class MdSnackBar {
     } else {
       snackBarRef.containerInstance.enter();
     }
+    if (config.autoHide > 0) {
```

Comment:
We should do `config.duration && cofig.duration > 0` in case someone sets it to null or undefined.